app/models.py

```diff
@@ -1179,6 +1213,7 @@ class Template(TemplateBase):
 
     service = db.relationship("Service", backref="templates")
     version = db.Column(db.Integer, default=0, nullable=False)
+    category = db.relationship("TemplateCategory", backref="templates")
```

Comment:
I would change this to template_category. Ignore my relationship comment above, this should work!